.azure/build-pipeline.yaml

```diff
@@ -64,7 +64,7 @@ jobs:
         env:
           DOCKER_USER: $(DOCKER_USER)
           DOCKER_PASS: $(DOCKER_PASS_SECRET)
-          MVN_ARGS: '-B'
+          MVN_ARGS: '-B -X -e'
```

Comment:
Is this still necessary?

Reply:
No, not really. I was trying to debug `ClusterOperatorTest` which is failing in most cases. There is some race condition with vertx I think.